test/test_foreach.py

```diff
@@ -217,6 +256,10 @@ def test_pointwise_op(self, device, dtype, op, is_fastpath):
             self._pointwise_test(
                 wrapped_op, ref, inputs, is_fastpath and not disable_fastpath, False, values=values, zero_size=zero_size
             )
+            with InplaceForeachVersionBumpCheck(self, sample.input):
```

Comment:
do we want to do the same thing we do for binary test here? e.g. move the context manager to be closer to where we actually call the op